### DISCUSSION TITLE
Welcome to Frog Discussions!

### DISCUSSION THREAD

**Discussion by amka** [TenderOwl/Frog#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by candiedoperation** [comment]
@amka, On clicking the **snip button**, using `GtkWindow.Hide()` on the main window would help taking screenshots easily instead of moving Frog's Window all over the desktop

**Reply by amka** [reply]
Sure, this is a part of the plans.

**Comment by vipranarayan14** [comment]
Hi @amka , this is a wonderful app. It's simple and very useful. Thanks!

What is the backend for the OCR, tesseract?

**Reply by amka** [reply]
Ah, yeah, the Frog uses tesseract.

**Comment by vipranarayan14** [comment]
@amka Great. The OCR results were good. So, I was curious. Thank you.

**Reply by amka** [reply]
I'm glad you're enjoying it.
